apps/examples/testcase/ta_tc/wifi_manager/itc/itc_wifimanager_main.c

```diff
@@ -886,16 +886,16 @@ static void itc_wifimanager_success_ratio_ap(void)
 
 	printf("\nSuccess Ratio \n");
 	if (init_cnt > 0) {
-		printf("[Initialization Wifi] Success Ratio = %.2f% \n", (s_init_cnt / init_cnt) * 100);
+		printf("[Initialization Wifi] Success Ratio = %.2f \n", (s_init_cnt / init_cnt) * 100);
```

Comment:
I think original `%` wants to represent the percentage, not the format specifier.
So you should modify to `%%`, not removing.

Reply:
yes, changed for percentage display